frontend/.eslintrc

```diff
@@ -129,7 +130,19 @@
       "error",
       { "allowSameFolder": true, "rootDir": "src", "prefix": "src" }
     ],
-    "no-else-return": ["error", { "allowElseIf": true }]
+    "no-else-return": ["error", { "allowElseIf": true }],
+    
+    "jest/expect-expect": "off",
```

Comment:
There are about three dozen linter errors with the default recommended ruleset in the current codebase; rules have been disabled for followup cleanup. Another alternative would be running codemod to suppress lint errors in test files to prevent adding more instances of errors for these rules. 

Let me know what is preferred from the team, if there is any reservation with the default recommended rules (https://github.com/jest-community/eslint-plugin-jest#rules)